app/util/fareUtils.js

```diff
@@ -119,6 +119,7 @@ export const shouldShowFarePurchaseInfo = (config, breakpoint, fares) => {
     !unknownFares &&
     fares?.length === 1 &&
     config.ticketPurchaseLink &&
+    config.ticketLinkOperatorCode &&
     breakpoint !== 'large'
```

Comment:
I wonder if this removes ticket button from hsl planner? If so, maybe add some dummy operator code like 'hsl' to its config as well.

There seems to be really many ticket related switches in the config. Let's avoid adding more if possible.

Reply:
It does! I forgot about hsl. I added a value for it now.